src/WooCommerce.php

```diff
@@ -1513,4 +1514,21 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
     return $dropdown_values;
   }
 
+    /**
+   * Invalidate the object cache in Woocommerce order status manager
+   * 
+   * @param int $post_id
+   * 
+   * @param \WP_Post $post
+   *   all post types - we're checking for wc_order_status post type
+   * 
+   * @return void
+   * 
+   */
+  private static function save_post_custom_status_manager(int $post_id, \WP_Post $post) {
```

Comment:
Let's make the function public. 

```suggestion
  public static function save_post_custom_status_manager(int $post_id, \WP_Post $post) {
```